pydoctor/epydoc2stan.py

```diff
@@ -975,6 +975,77 @@ def ensure_parsed_docstring(obj: model.Documentable) -> Optional[model.Documenta
     else:
         return None
 
+
+class _ParsedDocFromStan(ParsedDocstring):
+    """
+    A dummy L{ParsedDocstring}. 
+    
+    L{to_stan} method simply returns back what's given to L{_ParsedDocFromStan.__init__}. 
+    """
+    def __init__(self, stan: Tag):
+        super().__init__(fields=[])
+        self._fromstan = stan
+    def has_body(self) -> bool:
+        return True
+    def to_stan(self, docstring_linker: Any) -> Tag:
+        return self._fromstan
+    def to_node(self) -> Any:
+        raise NotImplementedError()
+
+def parsed_doc_from_stan(stan: Tag) -> 'ParsedDocstring':
+    """
+    Encapsulate the given L{Tag} into a L{ParsedDocstring}.
+    """
+    return _ParsedDocFromStan(stan)
+
+def ensure_parsed_summary(obj: model.Documentable) -> Tuple[Optional[model.Documentable], ParsedDocstring]:
```

Comment:
Maybe make it private and use a name that suggest this is a getter

I think the "ensure" part is not that relevant.

For the high-level function call, it's important to know that it extracts the summary from an `documentable`

What and how it does it and what it ensures, is low-level implementation.

```suggestion
def _get_parsed_summary(obj: model.Documentable) -> Tuple[Optional[model.Documentable], ParsedDocstring]:
```